.github/workflows/build-plugin-zip.yml

```diff
@@ -27,7 +27,6 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
         with:
-          ref: ${{ github.ref }}
```

Comment:
I think with this in place the zip we build is always from `trunk` instead of the branch. Can't test that without merging this in due to this PR being opened from a fork so if you download the artifact from this PR, none of the changes made here will be reflected in that